Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -133,6 +138,16 @@ public struct CustomerCenterConfigData {
                     return "Update"
                 case .updateWarningIgnore:
                     return "Continue"
+                case .platformMismatch:
+                    return "Platform mismatch"
+                case .pleaseContactSupportToManage:
+                    return "Please contact support to manage your subscription."
+                case .appleSubscriptionManage:
+                    return "You can manage your subscription via the App Store app on an Apple device."
```

Comment:
I would probably replace "via" with "by using" here, but just an opinion. But actually, can you with the app store app? I think this needs to be managed in Settings or via your AppleID somehow.

Reply:
Thanks for the feedback, I will update the text. You can actually manage the subscriptions in the app store by clicking the profile pic

Reply:
> You can actually manage the subscriptions in the app store by clicking the profile pic

TIL!